EnglishFilter/sections/general_elemhide.txt

```diff
@@ -97,6 +97,7 @@
 ##.div-gpt-ad:not([style^="width: 1px; height: 1px;"])
 ##[class^="div-gpt-ad"]:not([style^="width: 1px; height: 1px;"])
 !
+##div[class^="banner_"]
```

Comment:
This rule also can apply to many website.
https://www.bosa.co.kr/news/articleView.html?idxno=2154214
https://www.econovill.com/ ( https://www.econovill.com/news/articleView.html?idxno=540458 )
https://www.sisafocus.co.kr/news/articleView.html?idxno=262932
http://www.newsian.co.kr/ ( http://www.newsian.co.kr/news/articleView.html?idxno=49785 )
https://www.idsn.co.kr/ ( https://www.idsn.co.kr/news/articleView.html?idxno=50752 )
https://www.ggilbo.com ( https://www.ggilbo.com/news/articleView.html?idxno=858021 )
http://www.andongdaily.com/ ( http://www.andongdaily.com/news/articleView.html?idxno=24475 )
https://www.tiptipnews.co.kr/news/articleView.html?idxno=80869
http://www.fntoday.co.kr/ ( http://www.fntoday.co.kr/news/articleView.html?idxno=260316 )

Reply:
This rule is extremely dangerous) Please make domain specific.